oc-chef-pedant/spec/api/oc_id_spec.rb

```diff
@@ -2,6 +2,38 @@
 require 'openssl'
 
```

Comment:
:+1: can do. It makes sense to not only have that information in the Jira ticket, but also where you might _want to see_ it. 😉